[mercedesme] ConfigurationTest unstable

This test failed in: https://github.com/wborn/openhab-addons/actions/runs/3211692034/jobs/5250039131

```
[ERROR] Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.15 s <<< FAILURE! - in org.openhab.binding.mercedesme.ConfigurationTest
[ERROR] org.openhab.binding.mercedesme.ConfigurationTest.testCallbackUrl  Time elapsed: 0.059 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	at org.openhab.binding.mercedesme.ConfigurationTest.testCallbackUrl(ConfigurationTest.java:66)
```

--- a/bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/server/Utils.java
+++ b/bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/server/Utils.java
@@ -12,6 +12,7 @@
  */
 package org.openhab.binding.mercedesme.internal.server;
 
+import java.net.Inet4Address;
 import java.net.InetAddress;
 import java.net.NetworkInterface;
 import java.net.SocketException;
@@ -70,7 +71,9 @@ public class Utils {
                         for (Enumeration<InetAddress> addresses = iface.getInetAddresses(); addresses
                                 .hasMoreElements();) {
                             InetAddress address = addresses.nextElement();
-                            return address.getHostAddress();
+                            if (address instanceof Inet4Address) {
+                                return address.getHostAddress();
+                            }
                         }
                     }
                 }

--- a/bundles/org.openhab.binding.mercedesme/src/test/java/org/openhab/binding/mercedesme/ConfigurationTest.java
+++ b/bundles/org.openhab.binding.mercedesme/src/test/java/org/openhab/binding/mercedesme/ConfigurationTest.java
@@ -62,10 +62,11 @@ class ConfigurationTest {
     @Test
     public void testCallbackUrl() throws SocketException {
         String ip = Utils.getCallbackIP();
+        String message = "IP " + ip + " not reachable";
         try {
-            assertTrue(InetAddress.getByName(ip).isReachable(10));
+            assertTrue(InetAddress.getByName(ip).isReachable(10000), message);
         } catch (IOException e) {
-            assertTrue(false, "IP " + ip + " not reachable");
+            fail(message);
         }
     }
 }

